### ISSUE TITLE
Separating private messages to separate alerts popup

### ISSUE THREAD

**Issue by keithpickering** [MyAlerts/MyAlerts#1]
Hi, let me just say I absolutely love this plugin.

One thing I'd like to do: currently, all alerts are contained within a single popup. If I want to list private message alerts separately, along with their own number indicator and popup, how could I go about doing that?

**Comment by euantorano** [comment]
Hi,

Unfortunately that wouldn't be particularly easy at the minute. You'd need to add an additional variable and filter PM alerts off into it within the plugin.

**Comment by keithpickering** [comment]
Yeah, that's what I'm planning on doing, I just wondered if you had any pointers on getting started.

**Comment by euantorano** [comment]
Alright, it shouldn't be too difficult to do. I would keep most of the code as is, just split the alerts array before rendering the templates. You might also have to add a new template for the PM display, or you might be able to just use existing ones.
